### ISSUE TITLE
Start Minimized

### ISSUE THREAD

**Issue by maximzabuti** [troylar/quiet-on-the-set#19]
Hi,
Very nice tool but it would be even nicer if you could add an ability to start it minimized.
Thanks!

**Comment by 000A000** [comment]
Yeah, I also wanted to know how you would have this program minimised by default.
So when I start up my laptop, the program remains in my tray (still active though) and won't show up on my screen. 

Edit: I guess there is a reason this website is called **git**hub. 
If you modify the source code (after downloading or copying it into your repository), I could not find a way to compile it (turn it into an installer.exe) without first installing Microsoft Visual Studios (which I am not going to do). 
This program also has another issue, where it opens up multiple files each time the window is opened (also done on purpose). 



If anybody finds something like `https://codepen.io/` but for creating programs on Windows PC, do let me know.
`https://github.com/features/codespaces` You have to pay for that. 
`https://github.com/000A000` I can't even see all the comments I've done on this website by viewing my profile (this is my first comment on this website). 

`https://www.actualsolution.com/download/`
^That program does the job. However, you must pay for unlimited use (test out with the 14-day free trial). 
OR you could buy volume-limiting headphones (why I wanted this program, for headphone volume safety).
